tuf/data/types.go

```diff
@@ -108,7 +155,7 @@ type Signature struct {
 
 // Files is the map of paths to file meta container in targets and delegations
 // metadata files
-type Files map[string]FileMeta
+type Files map[RoleName]FileMeta
```

Comment:
The string in this case is a target name, not a role name. This change needs to be reverted.